src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/controller/QueueAdminIntTest.kt

```diff
@@ -1,10 +1,21 @@
 package uk.gov.justice.digital.hmpps.personrecord.controller
 
 import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.mock.mockito.MockBean
 import uk.gov.justice.digital.hmpps.personrecord.config.WebTestBase
+import uk.gov.justice.digital.hmpps.personrecord.health.PersonMatchHealthPing
+import uk.gov.justice.digital.hmpps.personrecord.health.PersonRecordHealthPing
 
 class QueueAdminIntTest() : WebTestBase() {
 
+  @MockBean
+  private lateinit var personMatchHealthPing: PersonMatchHealthPing
+
+  @MockBean
+  @Autowired
+  private lateinit var personRecordHealthPing: PersonRecordHealthPing
```

Comment:
i dont think we should define these on every integration test. Maybe move them in to the IntegrationTestBase so they are initialized on every test class